src/test/java/uk/nhs/prm/e2etests/test/ArchivedE2ETest.java

```diff
@@ -138,34 +139,36 @@ void shouldReceivingAndTrackAllLargeEhrFragments_DevAndTest() {
         assertTrue(oldTransferTrackerService.isStatusForConversationIdPresent(triggerMessage.getConversationId(), EHR_TRANSFER_TO_REPO_COMPLETE.status));
     }
 
-    @Disabled
-    @ParameterizedTest
-    @EnumSource(value = TemplateVariant.class, names = {"POSITIVE_ACKNOWLEDGEMENT", "NEGATIVE_ACKNOWLEDGEMENT"})
-    void shouldPutAcksOnMHSInboundAndUpdateEhrOutDbStatus(TemplateVariant templateVariant) {
-        // given
-        UUID ackMessageId = UUID.randomUUID();
-        String ackConversationId = UUID.randomUUID().toString();
-        String expectedTypeCode = templateVariant.name().equals("POSITIVE_ACKNOWLEDGEMENT") ? "AA" : "AR";
-
-        log.info("{} conversationId: {}, messageId: {}", templateVariant.name(), ackMessageId, ackConversationId);
-
-        String ackMessage = this.templatingService.getTemplatedString(templateVariant,
-                AcknowledgementTemplateContext.builder()
-                        .messageId(ackMessageId)
-                        .build());
-
-        // when
-        mhsInboundQueue.sendMessage(ackMessage, ackConversationId);
-
-        Acknowledgement acknowledgement = await().atMost(30, TimeUnit.SECONDS)
-                .with().pollInterval(2, TimeUnit.SECONDS)
-                .until(() -> ehrOutService.findAcknowledgementByMessageId(ackMessageId), Objects::nonNull) ;
-
-        // then
-        String actualTypeCode = acknowledgement.getAcknowledgementTypeCode();
-        assertThat(actualTypeCode).isEqualTo(expectedTypeCode);
-        log.info("The acknowledgement typeCode of {} is {}.", ackMessageId, actualTypeCode);
-    }
+    // TODO PRMT-4669 commented out due to UUID/STRING discrepancy - archived test, not worth fixing at the moment
```

Comment:
The ticket on this `TODO` is referring to this PR (PRMT-4669), therefore should be changed to another ticket to address this.